.travis.yml

```diff
@@ -10,4 +10,4 @@ before_script:
   - go install github.com/golangci/golangci-lint/cmd/golangci-lint
 script:
   - golangci-lint run --skip-files ".*.pb.go"
-  - go test -v ./... # go test -v -race ./... is failing, should be fixed
+  - go test -v -race ./...
```

Comment:
Move the race test inside the makefile. The prow job (travis is not used) launches `make test`